template/pyproject.toml.jinja

```diff
@@ -74,7 +77,7 @@ source = ["src/{{packagename}}"]
 
 [tool.github.info]
 organization = 'easyScience'
-repo = {{packagename}}Lib
+repo = "{{packagename}}"
```

Comment:
Should we make it possible to handle the `Lib` cases by adding a `repo_postfix` variable

Reply:
On the other hand, we might want this to be more generic. On the one hand, "EasyQens" will have both "Lib" and "App" but other projects like "EasyUtilities" won't have a `repo_prefix`, so maybe specifying the name directly is more sensible?